midealocal/devices/dc/__init__.py

```diff
@@ -81,8 +166,26 @@ def process_message(self, msg: bytes) -> dict[str, Any]:
         ]
         for status in self._attributes:
             if hasattr(message, str(status)):
+                value = getattr(message, str(status))
+                # parse progress
                 if status == DeviceAttributes.progress:
-                    self._attributes[status] = progress[getattr(message, str(status))]
+                    self._attributes[status] = progress[value]
+                # parse status
+                elif status == DeviceAttributes.status:
+                    if value in MideaDCDevice._status:
+                        self._attributes[DeviceAttributes.status] = (
+                            MideaDCDevice._status.get(value)
+                        )
+                    else:
+                        self._attributes[DeviceAttributes.status] = None
```

Comment:
```suggestion
                    self._attributes[DeviceAttributes.status] = (
                            MideaDCDevice._status.get(value)
                        )
```

The purpose of get() is exactly to avoid the if

Reply:
value may not list in _status